src/pages/workspaces/workspace/Workflows.test.js

```diff
@@ -0,0 +1,147 @@
+          url: 'http://agora.dsde-dev.broadinstitute.org/api/v1/methods/gatk/haplotypecaller-gvcf-gatk4/1',
+          synopsis: 'Runs HaplotypeCaller from GATK4 in GVCF mode on a single sample',
+          entityType: 'Workflow',
+          snapshotComment: '',
+          snapshotId: 1,
+          namespace: 'gatk',
+        },
+        {
+          name: 'unfeatured-workflow',
+          createDate: '2018-11-30T22:18:35Z',
+          url: 'http://agora.dsde-dev.broadinstitute.org/api/v1/methods/gatk/unfeatured-workflow/1',
+          synopsis: 'Not a featured workflow',
+          entityType: 'Workflow',
+          snapshotComment: '',
+          snapshotId: 1,
+          namespace: 'gatk',
+        },
+      ])
+    );
+    loadMethod = jest.fn(() => {
+      Promise.resolve({
+        managers: ['test@test.org'],
+        name: 'haplotypecaller-gvcf-gatk4',
+        createDate: '2018-11-30T22:18:35Z',
+        documentation: 'Some workflow documentation',
+        entityType: 'Workflow',
+        snapshotComment: '',
+        snapshotId: 1,
+        namespace: 'gatk',
+        payload: '## Some workflow content',
+        url: 'http://agora.dsde-dev.broadinstitute.org/api/v1/methods/gatk/haplotypecaller-gvcf-gatk4/1',
+        public: true,
+        synopsis: 'Runs HaplotypeCaller from GATK4 in GVCF mode on a single sample',
+      });
+    });
+    methodApi = () => ({
+      get: loadMethod,
+      configs: configurations,
+      toWorkspace,
+    });
+    configurations = jest.fn(() => Promise.resolve([methodConfiguration]));
+    toWorkspace = jest.fn(() => Promise.resolve());
+
+    Ajax.mockImplementation(() => ({
+      FirecloudBucket: {
+        getFeaturedMethods,
+      },
+      Methods: {
+        list: getRuns,
+        method: methodApi,
+      },
+      Metrics: {
+        captureEvent: jest.fn(() => {}),
+      },
+    }));
+  });
+
+  it('renders recommended workflows', async () => {
+    // Arrange
+    const props = { namespace, name, ws, onDismiss };
+
+    // Act
+    await act(async () => {
+      render(h(FindWorkflowModal, props));
+    });
+
+    expect(getFeaturedMethods).toHaveBeenCalled();
+    expect(getRuns).toHaveBeenCalled();
+
+    expect(screen.queryByText('haplotypecaller-gvcf-gatk4')).toBeInTheDocument();
+    expect(screen.queryByText('Runs HaplotypeCaller from GATK4 in GVCF mode on a single sample')).toBeInTheDocument();
+    expect(screen.queryByText('unfeatured-workflow')).not.toBeInTheDocument();
+    expect(screen.queryByText('Not a featured workflow')).not.toBeInTheDocument();
+  });
+
+  it('supports importing recommended workflows', async () => {
+    // Arrange
+    const user = userEvent.setup();
+
+    const props = { namespace, name, ws, onDismiss };
+
+    // Act
+    await act(async () => {
+      render(h(FindWorkflowModal, props));
+    });
+
+    const workflowCardButton = screen.getAllByRole('button').filter((button) => button.textContent.includes('haplotypecaller-gvcf-gatk4'))[0];
```

Comment:
I felt like there must surely be a neater way to do this, but I couldn't find it. If anyone has recommendations I'd love to hear them!